src/clipboard.js

```diff
@@ -168,3 +170,11 @@ export function cleanWhitespace (str) {
       : ' '
     ))
 }
+
+export function replaceAllQuotes (str) {
+  if (replaceQuotes.quotes) {
```

Comment:
Can you alsoadd `if (replaceQuotes &&)` here so we can deal with a completely undefined config?

Reply:
it's always an empty object https://github.com/livingdocsIO/editable.js/pull/212/files#diff-e4b7db0db2bb9f545c59492bc99ef442cd32bc5402335021236b8f159dd3cd06R22

Reply:
perfect!